.yamllint.yaml

```diff
@@ -37,7 +37,7 @@ rules:
 
   line-length:
     max: 100
-    allow-non-breakable-words: false
+    allow-non-breakable-words: true
```

Comment:
this has to be set as true; otherwise, the clp-config.yaml with the long URL inside would fail the rule.

@coderabbitai create an issue to track this lint rule change, and i will submit the lint rule change in a separate PR